distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -329,9 +326,6 @@ public String toString() {
   public static final DynamicUrl SERVICE_ROOT =
       new DynamicUrl(SECURE_ROOT, HTTPS_PORT, "/services");
 
-  public static final DynamicUrl INSECURE_SERVICE_ROOT =
```

Comment:
I think this is used downstream, is it necessary to remove?

Reply:
i can put that back.

Reply:
if it is used downstream, should that variable to added to the extend class rather? i did a quick search and don't see its usage tho.